### PR TITLE
ICNet with various sparsities

### PR DESCRIPTION
**Pull request by eldercrow**
Three models with various sparsities. This PR includes tiny YOLO v2, YOLO v2, ICNet thus the previous PR can be removed if this PR gets approved. Sorry for confusion.

### CODE REVIEW

**Comment by IRDonch** [comment]
I merged the YOLO stuff with #798, so please rebase.

**Comment by eldercrow** [comment]
I've already rebased it, could you please check?

File: models/intel/icnet-camvid-tf-ava-0001/description/icnet-camvid-tf-ava-0001.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+# icnet-camvid-tf-ava-0001
+
+## Use Case and High-Level Description
+
+A trained model of ICNet for fast semantic segmentation, trained on the CamVid dataset from scratch using the TensorFlow framework. For more details about the original floating point model, check out the [paper](https://arxiv.org/abs/1704.08545).
```
Comment (eizamaliev) [inline models/intel/icnet-camvid-tf-ava-0001/description/icnet-camvid-tf-ava-0001.md]: `CamVid\*`, `TensorFlow\*`, etc, hereinafter

File: models/intel/icnet-camvid-tf-ava-sparse-30-0001/description/icnet-camvid-tf-ava-sparse-30-0001.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
+# icnet-camvid-tf-ava-sparse-30-0001
+
+## Use Case and High-Level Description
+
+A trained model of ICNet for fast semantic segmentation, trained on the CamVid dataset from scratch using the TensorFlow framework. The trained model has 30% sparsity (ratio of 0's within all the covolution kernel weights). For more details about the original floating point model, check out the [paper](https://arxiv.org/abs/1704.08545).
```
Comment (eizamaliev) [inline models/intel/icnet-camvid-tf-ava-sparse-30-0001/description/icnet-camvid-tf-ava-sparse-30-0001.md]:
```suggestion
A trained model of ICNet for fast semantic segmentation, trained on the CamVid dataset from scratch using the TensorFlow framework. The trained model has 30% sparsity (ratio of 0's within all the convolution kernel weights). For more details about the original floating point model, check out the [paper](https://arxiv.org/abs/1704.08545).
```

File: models/intel/icnet-camvid-tf-ava-0001/description/icnet-camvid-tf-ava-0001.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+- `B` - batch size
+- `C` - channel
+- `H` - height
+- `W` - width
+
+Channel order is `BGR`
+
+## Output
+
+Semantic segmentation class prediction map, shape -`1,720,960`, output data format is `B,H,W` where:
```
Comment (eizamaliev) [inline models/intel/icnet-camvid-tf-ava-0001/description/icnet-camvid-tf-ava-0001.md]:
```suggestion
Semantic segmentation class prediction map, shape - `1,720,960`, output data format is `B,H,W` where:
```
hereinafter

File: models/intel/icnet-camvid-tf-ava-0001/description/icnet-camvid-tf-ava-0001.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+## Input
+
+Image, shape - `1,3,720,960`, format is `B,C,H,W` where:
+
+- `B` - batch size
+- `C` - channel
+- `H` - height
+- `W` - width
+
+Channel order is `BGR`
```
Comment (eizamaliev) [inline models/intel/icnet-camvid-tf-ava-0001/description/icnet-camvid-tf-ava-0001.md]:
```suggestion
Channel order is `BGR`.
```
hereinafter

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

File: models/intel/index.md [hunk 5]
```diff
@@ -110,6 +110,9 @@ detect areas with complex shape (for example, free space on the road).
 | [road-segmentation-adas-0001](./road-segmentation-adas-0001/description/road-segmentation-adas-0001.md)                                        | 4.770                | 0.184      |
 | [semantic-segmentation-adas-0001](./semantic-segmentation-adas-0001/description/semantic-segmentation-adas-0001.md)                            | 58.572               | 6.686      |
 | [unet-camvid-onnx-0001](./unet-camvid-onnx-0001/description/unet-camvid-onnx-0001.md)                                                          | 260.1                | 31.03      |
+| [icnet-camvid-ava-tf-0001](./icnet-camvid-ava-tf-0001/description/icnet-camvid-ava-tf-0001.md)                                                   | 151.82                | 25.45       |
```
Comment (eizamaliev) [inline models/intel/index.md]: Replace `ava-tf` with `tf-ava`

Comment (vladimir-dudnik) [reply models/intel/index.md]: fixed

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

File: models/intel/index.md (hunk 5 above)
Comment (vladimir-dudnik) [reply models/intel/index.md]: @eizamaliev why you request this change? Actual model name in internal infra ava-tf.

**Pull request merged**